go/mock_flightclient.go

```diff
@@ -0,0 +1,141 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
You should not commit this file, it should be generated before running the tests

Reply:
This is not generated automatically before running tests. I generated it manually by running a command using mockgen. To do it completely automatically we would need to create a script or add CI/CD to this. Let me know what you think.